build.gradle

```diff
@@ -683,6 +713,15 @@ project('standalone') {
         }
     }
 
+    configurations.testRuntimeClasspath {
+        resolutionStrategy {
+                force "com.google.inject.extensions:guice-assistedinject:" + guiceVersion
+                force "com.google.inject.extensions:guice-servlet:" + guiceVersion
+                force "com.google.inject:guice:" + guiceVersion
+                force "com.google.guava:guava:25.0-jre"
```

Comment:
Having hardcoded versions here is not a good idea. Normally we try to parameterize all these versions.